### ISSUE TITLE
Можно ли переводить видео с локального диска?

### ISSUE THREAD

**Issue by BigTolly** [sodapng/voice-over-translation#11]
В хроме нажать Ctrl+O и выбрать файл с видео.
В скрипт добавил: `*file:///*` но не сработало.

**Comment by ilyhalight** [comment]
> В хроме нажать Ctrl+O и выбрать файл с видео. В скрипт добавил: `*file:///*` но не сработало.

Нет, перевод доступен, только, на определенных сайтах, которые находятся в вайтлисте у Яндекса
